src/index.rs

```diff
@@ -443,12 +445,14 @@ impl PouIndexEntry {
         pou_name: &str,
         linkage: LinkageType,
         location: SymbolLocation,
+        super_class: Option<String>,
```

Comment:
pass a &str instead of string.
You can use Option<&str> and use .as_ref to get that option